### PR TITLE
first stage

### PR DESCRIPTION
**Pull request by Fmarvelsk**
My PR

### CODE REVIEW

File: scripts/myscript.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
-function sayHello(fullname, id, language) {
-  console.log(
-    `Hello World, this is ${fullname} with HNGi7 ${id} using ${language}`
-  );
-}
-sayHello();
+function sayHello(fullname, id, language) {
```
Comment (NdubuisiJr) [inline scripts/myscript.js:1]: Look at this properly. You didn't pass the arguments you specified in the `sayHello` function

**Comment by NdubuisiJr** [comment]
Please, the name of your file should be unique.

File: scripts/myscript.js (hunk 1 above)
Comment (Fmarvelsk) [reply scripts/myscript.js:1]: that not mine

**Comment by Fmarvelsk** [comment]
fixed

**Pull request closed without merging**